docs/ai/reference_http.rst

```diff
@@ -0,0 +1,382 @@
+.. _ref_ai_http_reference:
+
+=====================
+AI HTTP API Reference
+=====================
+
+:edb-alt-title: AI Extension HTTP API
+
+.. note::
+
+    All |Gel| server HTTP endpoints require :ref:`authentication
+    <ref_http_auth>`, such as `HTTP Basic Authentication
+    <https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#basic_authentication_scheme>`_
+    with Gel username and password.
+
+
+Embeddings
+==========
+
+``POST``: ``https://<gel-host>:<port>/branch/<branch-name>/ai/embeddings``
+
+Generates text embeddings using the specified embeddings model.
+
+
+Request headers
+---------------
+
+* ``Content-Type: application/json`` (required)
+
+
+Request body
+------------
+
+.. code-block:: json
+
+    {
+      "model": string,        // Required: Name of the embedding model
+      "inputs": string[],     // Required: Array of texts to embed
+      "dimensions": number,   // Optional: Number of dimensions to truncate to
+      "user": string          // Optional: User identifier
+    }
```

Comment:
I don't _think_ we do a syntax check pass on JSON, but this isn't valid JSON, so if someone copy-pasted it, it wouldn't work.

Maybe you're meaning to use TypeScript to define this interface? I personally wouldn't bother doing that, and instead just list the properties like you do below.